src/pdf/pdf-view.js

```diff
@@ -827,10 +871,12 @@ class PDFView {
 
 	navigateToNextPage() {
 		this._iframeWindow.PDFViewerApplication.pdfViewer.nextPage();
+		this.a11yWillPlaceVirtCursorOnTop();
```

Comment:
[_updateViewStats](https://github.com/zotero/reader/blob/8ae110505de4de25df81954e15c5fd94f5cf2d17/src/pdf/pdf-view.js#L2394) would be a better place to track whenever page changes, because there are many ways how page number can be changed.

Reply:
I suppose the question is if we always want to focus a node on the when the page changes or only in some cases. For instance, if one just scrolls the page normally with a mouse, we don't want to do that. In fact, the virtual cursor target is cleared on scroll to avoid jumping to the focused element on mouse click into the document. I'll try out what will happen if we move it to `updateViewStats`!

Reply:
Yeah, it was a great call! Doing it through `_updateViewStats` allows us to have pretty just one place where the target is updated after any kind of scrolling, instead of having it scattered across multiple methods. I refactored it a bit to run through a debounce to only perform the update once when all the scrolling is finished.